tsv-tennis-app/src/index.css

```
@@ -1 +1,117 @@
+  --radius-sm: 0px;
+  --radius-md: 0px;
+  --radius-lg: 0px;
+  --radius-xl: 0px;
+  --radius-2xl: 0px;
+  --radius-3xl: 0px;
+  --radius-4xl: 0px;
+  /* --radius-full intentionally left for avatars/pills */
+}
+
+@font-face {
+  font-family: "Archivo";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Archivo" (font-family-name-quotes)

(font-family-name-quotes)